dev/com.ibm.ws.cdi.jee_fat/publish/servers/cdi40SessionPersistenceServer/server.xml

```diff
@@ -0,0 +1,34 @@
+<server description="Server for testing Session Persistence with Faces/CDI ">
```

Comment:
Missing copyright here

Reply:
The FAT here doesn't have copyrights here, so I'm excluding it too.